src/Microsoft.Health.Fhir.Api/Features/Health/StorageInitializedHealthCheck.cs

```diff
@@ -34,6 +44,13 @@ public Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, Canc
                 return Task.FromResult(new HealthCheckResult(HealthStatus.Degraded, $"Storage is initializing. Waited: {(int)waited.TotalSeconds}s."));
             }
 
+            // Check if Storage customer-managed key (CMK) is properly set.
```

Comment:
I would rephrase this comment to "Check if customer-managed key (CMK) is properly set.".